app/controllers/api/v1_controller.rb

```diff
@@ -14,8 +14,9 @@ class Api::V1Controller < ApiController
   protected
 
   def authenticate
-    @current_user = token_auth if params[:auth_token]
+    @current_user = token_auth
```

Comment:
@robacarp This file is the only real big change, as the other controllers and views are just basically copy-pasta from doorkeeper gem to customize them for tb's `require_login!` and templates